### ISSUE TITLE
`function-call-argument-newline`:  change default from "always" to "consistent"

### ISSUE THREAD

**Issue by vladshcherbin** [eslint-stylistic/eslint-stylistic#1261]
### Clear and concise description of the problem

`function-call-argument-newline` defaults to `"always"`, so the bare rule flags the most common call shape:

```ts
readFile(path, 'utf8')
// error: There should be a line break after this argument.
```

I can't name a style that puts every argument of every call on its own line. In practice [everyone](https://github.com/search?q=%22%40stylistic%2Ffunction-call-argument-newline%22&type=code) who enables the rule writes `["error", "consistent"]` first, which makes the bare form a trap.

### Suggested solution

Default to `"consistent"`. The rule keeps its point: mixed layouts like

```ts
fn(a,
  b, c)
```

still get flagged, while both normal layouts pass, all arguments on one line or one per line.

### Additional context

v6 is in beta. A default change fits a major, and if it misses this one it waits a full cycle.

### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Contributes

- [x] If this feature request is accepted, I am willing to submit a PR to fix this issue

**Comment by 9romise** [comment]
Thanks for the proposal. We’re going to close this PR because the affected rule is being superseded by `@stylistic/list-style`, which is now stable in v6.
The legacy rule will be deprecated and behaviorally frozen, so changing its default would add a new behavior change to a rule users are being asked to migrate away from. If the desired behavior is still needed, please propose it as an improvement to `list-style` instead.
